core/color_names.inc

```diff
@@ -0,0 +1,1228 @@
+// Names from https://en.wikipedia.org/wiki/List_of_colors (through https://raw.githubusercontent.com/SuperUserNameMan/color_to_name/616a7cddafefda91478b7bc26167de97fb5badb1/godot_version.gd), slightly edited and normalized
+static Map<String, Color> _named_colors;
+static void _populate_named_colors() {
+	if(!_named_colors.empty()) return;
+	_named_colors.insert("absolutezero", Color(0.00, 0.28, 0.73 ));
```

Comment:
I also agree that only the HTML colors should be kept. The intention is to make it more user-friendly. Using `ColorN('fuzzywuzzy')` won't help with that.